steward/src/commands.rs

```diff
@@ -78,13 +78,21 @@ impl Configurable<StewardConfig> for EntryPoint {
             .map(PathBuf::from)
             .unwrap_or_else(|| CONFIG_FILE.into());
 
-        if filename.exists() {
+        let config_env_variable = std::env::var("VARIABLE").is_ok();
```

Comment:
We shouldn't use `VARIABLE` as our env var name. Let's go with `STEWARD_CONFIG`.

Also, calling `is_ok()` is telling you if the `Result` from `std::env::var()` is `Ok`. In order to retrieve the variable, we'd need to assign `std::env::var("STEWARD_CONFIG")` to a variable, check if it `is_ok()`, and then unwrap it to get the value stored in the env variable.